src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
   return (
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires  `App` to be class-based component. Kindly do that to comply with project requirements.

Reply:
I didn't even notice that by default React.js, makes the App component Functional!

Thanks for the tip! It's solved!